lib/openstack/openstack_handle/handle.rb

```diff
@@ -25,16 +25,19 @@ class Handle
       "Orchestration" => :orchestration
     }
 
+    def self.try_connection_with_ssl
+      # attempt to connect without SSL
+      yield "http", {}
```

Comment:
Yeah.  :disappointed:

Good point that we really need both!  We need to know whether it's ssl and whether we can rely on ssh key verification on the appliance for that EMS.  For the verify setting, the user would have had to take extra action (import CA into appliance)

In fact, importing Provider CAs should be an option at some point.